### ISSUE TITLE
Sort order bug with nested blocks

### ISSUE THREAD

**Issue by eugenet8k** [rossPatton/stylint#237]
This piece of stylus code:

```stylus
+foo()
  helper(width)
    // my comment
    border width
    color red
```

Gives persistent alphabetical sorting warning. Even considering that c goes after b. If the comment removed it works fine. If nesting is removed or mixin is replaced with a class it works fine. The problem appears in this particular combination.

```
Warning: prefer alphabetical when sorting properties
File: style.styl
Line: 4: border width

Warning: prefer alphabetical when sorting properties
File: style.styl.styl
Line: 5: color red
```

.stylintrc
```json
{
  "blocks": false,
  "brackets": "true",
  "colons": "never",
  "colors": "always",
  "commaSpace": "always",
  "commentSpace": "always",
  "cssLiteral": "never",
  "depthLimit": false,
  "duplicates": true,
  "efficient": "always",
  "extendPref": "@extend",
  "globalDupe": true,
  "indentPref": 2,
  "leadingZero": "never",
  "maxErrors": false,
  "maxWarnings": false,
  "mixed": false,
  "namingConvention": false,
  "namingConventionStrict": false,
  "none": "never",
  "noImportant": false,
  "parenSpace": false,
  "placeholders": "always",
  "prefixVarsWithDollar": false,
  "quotePref": false,
  "semicolons": "never",
  "sortOrder": "alphabetical",
  "stackedProperties": "never",
  "trailingWhitespace": "never",
  "universal": false,
  "valid": true,
  "zeroUnits": "never",
  "zIndexNormalize": false
}
```

Stylint version: 1.3.5

**Comment by scottyschup** [comment]
Not quite the same (but related, and probably addressing one will fix the other) the same warning appears with nested rules:

```stylus
svg
  height 100%
  position absolute
  width 100%
  text
    font-size 50%
```

Since that ability to create nested rules is one of the strengths of stylus, it seems like this use case should be accounted for.

**Comment by fdeneux** [comment]
Also receiving sort order warnings on the following block:

```stylus
a.button,
button,
input[type="button"],
input[type="reset"],
input[type="submit"]
    -webkit-font-smoothing: antialiased
    appearance: none
    background-color: $color-primary
    border-radius: $border-radius-base
    border: 0
    color: $color-white
    cursor: pointer
    disable-mouse-outline()
    display: inline-block
    font-family: $font-family-base
    font-size: $font-size-base
    font-weight: 600
    line-height: 1
    margin: 0 0 $spacing-small
    padding: $spacing-small $spacing-base
    text-align: center
    text-decoration: none
    transition: background-color $animation-duration-base $animation-timing-base
    user-select: none
    vertical-align: middle
    white-space: nowrap
```

```bash
resources/assets/styles/base/buttons.styl
  line 10  -  prefer alphabetical when sorting properties sortOrder  border: 0
  line 11  -  prefer alphabetical when sorting properties sortOrder  color: $color-white
  line 12  -  prefer alphabetical when sorting properties sortOrder  cursor: pointer
  line 14  -  prefer alphabetical when sorting properties sortOrder  display: inline-block
  line 15  -  prefer alphabetical when sorting properties sortOrder  font-family: $font-family-base
  line 16  -  prefer alphabetical when sorting properties sortOrder  font-size: $font-size-base
  line 17  -  prefer alphabetical when sorting properties sortOrder  font-weight: 600
  line 18  -  prefer alphabetical when sorting properties sortOrder  line-height: 1
  line 19  -  prefer alphabetical when sorting properties sortOrder  margin: 0 0 $spacing-small
  line 20  -  prefer alphabetical when sorting properties sortOrder  padding: $spacing-small $spacing-base
  line 21  -  prefer alphabetical when sorting properties sortOrder  text-align: center
  line 22  -  prefer alphabetical when sorting properties sortOrder  text-decoration: none
  line 23  -  prefer alphabetical when sorting properties sortOrder  transition: background-color $animation-duration-base $animation-timing-base
  line 24  -  prefer alphabetical when sorting properties sortOrder  user-select: none
  line 25  -  prefer alphabetical when sorting properties sortOrder  vertical-align: middle
  line 26  -  prefer alphabetical when sorting properties sortOrder  white-space: nowrap

  ⚠  16 warnings
```

**Comment by jcohlmeyer** [comment]
I had this same issue but it seems like the warning stops if you make use of `depthLimit` with nested rules.
